### PR TITLE
Add README

### LINKED ISSUE #6: Add simple README
### Description

Add simple README to give an overview over the packages in the `kwiz-dotfiles` repo.

### Tasks

- [x] create and add README

### PR DESCRIPTION
**Pull request by nikzaugg**
### Description

This PR adds a README file for the whole repository.

### Related Issues

Issue #6 

### Checklist

- [x] Have you added tests where necessary? Do all the test pass? 
- [x] Have you added descriptive comments to your code?
- [x] Have you updated the documentation related to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# kwiz-dotfiles
+
+This repository holds the configuration files for `prettier` and `esling` that should be used in every project.
```
Comment (rschlaefli) [inline README.md]:
```suggestion
This repository holds the configuration files for `prettier` and `eslint` that should be used in every project.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## General
+
+**Note:** make sure you have a `.npmrc` with the following content:
```
Comment (rschlaefli) [inline README.md:7]: Also mention that one needs to login with a Github token locally.

**Review by rschlaefli — APPROVED** [review]
Thanks. Some minor additions, but otherwise LGTM.

File: README.md (hunk 2 above)
Comment (nikzaugg) [reply README.md:7]: done, thanks!

**Pull request merged**